indexer/reindex_bags_backlog.py

```diff
@@ -243,20 +243,35 @@ def publish(env, dry_run, role_arn):
 
 @click.command()
 @click.option("--env", default="stage", help="Environment to run against (prod|stage)")
+@click.option(
+    "--republish", default=False, is_flag=True, help="If not indexed, republish"
+)
 @click.option(
     "--role_arn", default=ROLE_ARN, help="AWS Role ARN to run this script with"
 )
-def confirm(env, role_arn):
+def confirm(env, republish, role_arn):
     config = get_config(env)
 
     dynamodb_client = create_client("dynamodb", role_arn)
     elastic_client = create_elastic_client(role_arn, ES_SECRETS)
+    sns_client = create_client("sns", role_arn)
 
     bags_to_publish = get_latest_bags(dynamodb_client, config["table_name"])
     bags_to_confirm = [key for (key, value) in bags_to_publish.items()]
 
     not_indexed = confirm_indexed(elastic_client, bags_to_confirm, config["es_index"])
-    print(f"NOT INDEXED: {not_indexed}")
+    if len(not_indexed) > 0:
```

Comment:
nit: in Python, collections are implicitly True if non-empty, implicitly False if not. It would be more idiomatic to write:

```suggestion
    if not_indexed:
```